zeebe/dynamic-config/src/test/java/io/camunda/zeebe/dynamic/config/ClusterConfigurationModifierTest.java

```diff
@@ -215,10 +221,95 @@ private static Arguments exporterAdded() {
           Named.of(
               "New Exporters Added",
               new ExporterConfigParameter(
-                  Set.of("expA", "expB", "exporter1", "exporter2"), expectedConfig)));
+                  withTwoEnabledExporters(),
+                  Set.of("expA", "expB", "exporter1", "exporter2"),
+                  expectedConfig)));
+    }
+
+    private static Arguments exporterReadded() {
+      final var initialConfig =
+          new DynamicPartitionConfig(
+              new ExportersConfig(
+                  Map.of(
+                      "expA",
+                      new ExporterState(0, State.ENABLED, Optional.empty()),
+                      "expC",
+                      new ExporterState(1, State.CONFIG_NOT_FOUND, Optional.empty()))));
+
+      final ClusterConfiguration currentConfiguration =
+          ClusterConfiguration.init()
+              .addMember(
+                  MemberId.from("0"),
+                  MemberState.initializeAsActive(
+                      Map.of(
+                          1,
+                          PartitionState.active(1, initialConfig),
+                          2,
+                          PartitionState.active(2, initialConfig))));
+
+      final var expectedConfig =
+          new DynamicPartitionConfig(
+              new ExportersConfig(
+                  Map.of(
+                      "expA",
+                      new ExporterState(0, State.ENABLED, Optional.empty()),
+                      "expC",
+                      new ExporterState(1, State.ENABLED, Optional.empty()))));
+      return Arguments.of(
+          Named.of(
+              "Exporters Readded",
+              new ExporterConfigParameter(
+                  currentConfiguration, Set.of("expA", "expC"), expectedConfig)));
+    }
+
+    private static Arguments disabledExporterRemoved() {
```

Comment:
🔧 method name doesn't match the scenario: disabled exporter without config stays disabled.